draft-ietf-acme-acme.md

```diff
@@ -1067,9 +1069,8 @@ field.  This can be used to an ACME account with an existing account in a
 non-ACME system, such as a CA customer database.
 
 To enable ACME account binding, a CA needs to provision the ACME client with a
-MAC key and a key identifier.  The key identifier MUST be an ASCII string.  The
```

Comment:
This should be covered by the new normative language in the top-level `Character Encoding` section since ASCII is a subset of UTF-8 (technically).